rdr_service/tools/tool_libs/backfill_physical_measurement_observation_code_text.py

```diff
@@ -0,0 +1,58 @@
+from datetime import datetime
+
+from rdr_service.dao.physical_measurements_dao import _OBSERVATION_RESOURCE_TYPE, PhysicalMeasurementsDao
+from rdr_service.lib_fhir.fhirclient_1_0_6.models import observation as fhir_observation
+from rdr_service.model.measurements import Measurement, PhysicalMeasurements
+from rdr_service.tools.tool_libs.tool_base import cli_run, ToolBase, logger
+
+tool_cmd = 'measurement-back-fill'
+tool_desc = 'back fill the valueCodeDescription for measurements'
+
+
+class MeasurementBackFill(ToolBase):
+    def run(self):
+        super(MeasurementBackFill, self).run()
+
+        latest_id = -10
+        with self.get_session() as session:
+            found_measurements = True
+            while found_measurements:
+                found_measurements = False
+                physical_measurements_query = session.query(
+                    PhysicalMeasurements.physicalMeasurementsId,
+                    PhysicalMeasurements.resource
+                ).filter(
+                    PhysicalMeasurements.physicalMeasurementsId > latest_id
+                ).order_by(PhysicalMeasurements.physicalMeasurementsId).limit(500)
```

Comment:
What is the range for the `physicalMeasurementsId`?

Reply:
Looks like it might be randomly generated and ranges between 100,000,000 and 1,000,000,000 right now.